onnxruntime/core/providers/armnn/activation/activations.cc

```diff
@@ -0,0 +1,109 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Copyright (c) 2020, NXP Semiconductor, Inc. All rights reserved.
+// Licensed under the MIT License
+
+#ifdef _WIN32
+#pragma warning(disable : 4244)
+#endif
+#include <thread>
```

Comment:
Why are you double including these(and a few more header files below)? They have already been included in activations.h which you have included here.

Reply:
will take a look and remove them